From: Jiri Slaby <jslaby@suse.cz>
Subject: syzkaller: no need for gcc plugins
Patch-mainline: never, only hack until we have gcc-devel pkg
References: bnc#1000094

We have no support in gcc in Factory yet. Until gcc-devel is in
factory, we have to work around the need for GCC_PLUGINS.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 lib/Kconfig.debug |    2 --
 1 file changed, 2 deletions(-)

--- a/lib/Kconfig.debug
+++ b/lib/Kconfig.debug
@@ -727,8 +727,6 @@ config KCOV
 	bool "Code coverage for fuzzing"
 	depends on ARCH_HAS_KCOV
 	select DEBUG_FS
-	select GCC_PLUGINS if !COMPILE_TEST
-	select GCC_PLUGIN_SANCOV if !COMPILE_TEST
 	help
 	  KCOV exposes kernel code coverage information in a form suitable
 	  for coverage-guided fuzzing (randomized testing).
